.circleci/config.yml

```diff
@@ -26,8 +26,8 @@ jobs:
           name: Test
           command: make test TESTREPORT=test-results/go/results.xml
       - run:
-          name: Code Quality
-          command: make lint
+          name: Install golangci-lint and check code Quality
+          command: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1 && make lint
```

Comment:
let's not :)

we can download the script and commit it in a scripts/ dir, and then call it from here. at least we have an audit record of what it did.

also, why not add the download to "make dep"?

Reply:
Added to  `scripts` directory. changed `make lint` to install golangci-lint.